### PR TITLE
 Add archive method

### PR DESCRIPTION
**Pull request by nashousni**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -68,7 +68,7 @@ usage() {
 writeSetting() {
     if [[ $# -lt 2 || $# -gt 3 ]]; then
         echo "WARNING: writeSetting <setting> <value> [comment (true|false)]"
-        exit 1
+        exit 1CMake
```
Comment (geofjamg) [inline install.sh]: ????

File: afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/AppStorage.java [hunk 2]
```diff
@@ -32,6 +32,10 @@
 
     boolean isRemote();
 
+    default boolean isLocalSystem() {
```
Comment (geofjamg) [inline afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/AppStorage.java]: What is the purpose of adding this? 
It seems to be a bad design as is add implementation info in interface.

**Comment by pl-buiquang** [comment]
Why is there so many commits unrelated to this feature here ?
Can you clean up your branch ?

**Comment by geofjamg** [comment]
> Why is there so many commits unrelated to this feature here ?
> Can you clean up your branch ?

We will squash when merging to master so it is not really an issue.

**Comment by nashousni** [comment]
> Add unit test to reach Sonar quality gateway
updated

**Pull request merged**